src/Serializer/Normalizer/PhoneNumberNormalizer.php

```diff
@@ -101,6 +101,14 @@ public function denormalize($data, $class, $format = null, array $context = []):
      */
     public function supportsDenormalization($data, $type, $format = null, array $context = []): bool
     {
-        return 'libphonenumber\PhoneNumber' === $type && \is_string($data);
+        return PhoneNumber::class === $type && \is_string($data);
+    }
+
+    /**
+     * for symfony/serializer >= 6.3.
+     */
+    public function getSupportedTypes(?string $format): array
+    {
+        return [PhoneNumber::class => false];
```

Comment:
I used `false` to indicate that we cannot cache the calls to `supportsDenormalization` because there we also check `is_string($data)`